src/main/scala/Adapter.scala

```diff
@@ -14,6 +14,8 @@ import rocketchip._
 import rocket.XLen
 import cde.{Parameters, Field}
 
+import chisel3.core.ExplicitCompileOptions.NotStrict
```

Comment:
@jackkoenig does it make sense that I need this? It says `Not bound to synthesizable node, currently only Type description` on line 210.